src/extension.ts

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+import {
+  ExtensionContext,
+  languages,
+  CompletionItemProvider,
+  CompletionItem,
+  CompletionItemKind,
+  ProviderResult,
+  Disposable,
+} from 'vscode'
+
+const snippets = require('../snippets/snippets.json')
+
+interface Snippet {
+  prefix: string
+  body: string
+  description: string
+}
+
+let registerCompletionProvider: Disposable | undefined
+
+class PSCompletionItem extends CompletionItem {
+  constructor(snippet: Snippet) {
+    super('psr', CompletionItemKind.Snippet)
+  }
```

Comment:
I tried to add a snippet body here, but without success. What else has to be done for the sample snippet from `snippets.json` to be registered?